projects/addon-mobile/components/pull-to-refresh/pull-to-refresh.template.html

```diff
@@ -1,6 +1,7 @@
 <ng-container *polymorpheusOutlet="component; context: {$implicit: (pulling$ | async) || 0}"></ng-container>
 
 <div
+    [style.position]="'relative'"
```

Comment:
nit: maybe move it to class?

Reply:
There was no class here at all, I thought it might be better to have 1 line, instead of 1 line to assign class + 3 lines in LESS to define it 🙂

Reply:
I thought it is for increased the specificity